include/seqan3/alignment/aligned_sequence/aligned_sequence_concept.hpp

```diff
@@ -64,14 +64,14 @@ struct unaligned_seq<t>
 };
 
 // customisation point for our gap decorators.
-//!\brief Exposes the unaligned sequence type if *t* exposes the type member `unaligned_seq_type`.
+//!\brief Exposes the unaligned sequence type if *t* exposes the type member `unaligned_sequence_type`.
 template <typename t>
 //!\cond
-    requires requires { typename std::remove_reference_t<t>::unaligned_seq_type; }
+    requires requires { typename std::remove_reference_t<t>::unaligned_sequence_type; }
 //!\endcond
 struct unaligned_seq<t>
```

Comment:
the name of this type trait can stay as is because it is detail (in contrast to the `template <typename unaligned_seq_t>` in gap_decorator)

Reply:
ok that makes sense. thx :)